README.md

```diff
@@ -21,6 +21,13 @@ actually implemented, can instead be obtained by from [the latest
 release](https://github.com/tezos-commons/homebase-lite/releases/latest)
 or produced with [the `homebase-lite` tool](docs/tool.md#getting-the-contract-documentation).
 
+## Using [TZIP-16](https://tzip.tezosagora.org/proposal/tzip-16/) off-chain views
+
+The contract declares TZIP-16 metadata in its storage, which includes two off-chain views:
+
+- `currentConfig`: accepts no parameters and returns the [`Configuration`](https://github.com/tezos-commons/homebase-lite/blob/autodoc/master/documentation.md#types-Configuration) object containing the current configuration of the contract (as defined in the spec).
+- `proposalInfo`: accepts a proposal [`URI`](https://github.com/tezos-commons/homebase-lite/blob/autodoc/master/documentation.md#types-URI) (as `string`) and returns a [`ProposalInfo`](https://github.com/tezos-commons/homebase-lite/blob/autodoc/master/documentation.md#types-ProposalInfo) object for the proposal identified by the `URI`, or throws a [`NoSuchProposal`](https://github.com/tezos-commons/homebase-lite/blob/autodoc/master/documentation.md#errors-NoSuchProposal) error if no such proposal exists.
+
```

Comment:
After thinking a bit I'm not sure if there's a better place to put this info in, but I think it that perhaps the header:
1. should be nested one more level
2. should not say "Using" as... there's nothing really about instructions here

However a larger change that I want to propose is to allow _some of_ the info in the metadata to be changed in the tool CLI from the default ones (which could be the ones that are currently used).
For example it's very likely that someone will want to change the `name` of a specific instance of this contract that they deploy and it would be nice to let them do that as easily as everything else :slightly_smiling_face:

Reply:
> some

I really couldn't think of anything but name and description. Anything else should imply changes to the source code.

Reply:
Fair enough, I didn't think about that but I agree :+1: